packages/@aws-cdk/cfnspec/package.json

```diff
@@ -19,11 +19,11 @@
   "main": "lib/index.js",
   "types": "lib/index.d.ts",
   "devDependencies": {
-    "@types/fs-extra": "^4.0.8",
+    "@types/fs-extra": "^5.0.4",
```

Comment:
Don't mix this PR with anything else...

Reply:
Yeah - made a force-push with just the new filter.